reference-manual/src/main/asciidoc/appendix.adoc

```diff
@@ -283,6 +283,133 @@ The URL used for phone home requests is
 http://phonehome.hazelcast.com/ping
 ----
 
+[appendix]
```

Comment:
You should apply line wrapping at 72 throughout.

Reply:
Ok, will do. But curious: why?

Reply:
It's the standard convention in text files, makes it a lot more readable if you don't have word wrapping on. I use rewrap plugin for VSCode.